app/static/js/admin/guest/explore/event-box-component.js

```diff
@@ -22,7 +22,12 @@ Vue.component('event-box', {
             return moment(this.event.start_time).format('ddd, MMM DD HH:mm A');
         },
         locationName: function () {
-            return this.event.location_name.split(",")[0];
+            if(this.event.location_name){
```

Comment:
split works well with non-empty string,and since we are retrieving from model and we defined the attribute location_name as a String and initialising it to None , there is no choice for it to be returned as undefined. @SaptakS 
![screenshot from 2016-12-24 12-22-18](https://cloud.githubusercontent.com/assets/15216503/21465699/f27dcc42-c9d3-11e6-9fa9-e47554257bae.png)